<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FaceBrick</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="statusText">
        <property name="inputMethodHints">
         <set>Qt::ImhNoAutoUppercase</set>
        </property>
        <property name="maxLength">
         <number>420</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateStatusButton">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListView" name="postsListView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>4</verstretch>
       </sizepolicy>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Synchronise"/>
    <addaction name="action_Logout"/>
    <addaction name="actionS_ettings"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <action name="action_Logout">
   <property name="text">
    <string>&amp;Logout</string>
   </property>
  </action>
  <action name="action_Synchronise">
   <property name="text">
    <string>&amp;Synchronise</string>
   </property>
  </action>
  <action name="actionS_ettings">
   <property name="text">
    <string>S&amp;ettings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
